set.mm

```diff
@@ -21469,9 +21467,16 @@ proposition with a distinct variable (closed form of ~ nfsb4 ).
 
   ${
     $d y z $.  $d x y $.
-    $( Lemma for ~ elsb4 , adapted from ~ equsb3lem .  (Contributed by Wolf
-       Lammen, 28-Jul-2022.) $)
-    elsb4lem $p |- ( [ x / y ] z e. y <-> z e. x ) $=
+    $( Specialization of ~ elsb4 with an extra distinct variable condition, but
+       no dependency on ~ ax-13 .  Adapted from ~ equsb3v .  (Contributed by
+       Wolf Lammen, 28-Jul-2022.)  (Revised by Steven Nguyen, 23-Jan-2023.) $)
+    elsb4v $p |- ( [ x / y ] z e. y <-> z e. x ) $=
+      ( wel nfv elequ2 sbiev ) CBDCADZBAHBEBACFG $.
+
+    $( Obsolete version of ~ elsb4v as of 23-Jan-2023.  (Contributed by Wolf
+       Lammen, 28-Jul-2022.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    elsb4vOLD $p |- ( [ x / y ] z e. y <-> z e. x ) $=
```

Comment:
To check with @wlammen but to me, since in elsb3v, elsb4v the only change was to replace sbie with sbiev, the OLD versions need not be kept (and when I do this kind of cleaning, I do not even put a "revised" tag, but you can do it if you want)).

Reply:
`If a proof, definition, axiom, or theorem is going to be removed, we often stage that change by first renaming its label with an OLD suffix`  From the conventions page.  The `often` allows you to not keep the OLD version, but if you like, you can do so.  I do not keep the OLD version in case of simple minimization.  sbiev was introduced to remove ax-13 from equsb3v, so this was a different case.